.rubocop.yml

```diff
@@ -14,6 +14,9 @@ AllCops:
 Bundler/DuplicatedGem:
   Enabled: false
 
+Layout/AlignArguments:
+  Enabled: false
```

Comment:
This cop results in 700+ changed lines. I agree with the cop in principle, but because we had made some weird indentation decisions in the past it's autocorrect tries to work with that and shoves everything over to the right.

i.e. something like this
```
ThingDoer.new(arg1,
  arg2,
  arg3
)
```

becomes
```
ThingDoer.new(arg1,
              arg2,
              arg3
)
```

when ideally (IMO) it would have been like this all along

```
ThingDoer.new(
  arg1,
  arg2,
  arg3
)
```

Wanted to discuss it with the team first.

Reply:
if rubymine had better rubocop integration I'd want this on, but since we only run it before committing, I think this is a wash and you've done the right thing in disabling it.

Reply:
Couldn't have said it better.